tooltester/src/main/java/io/dockstore/tooltester/client/cli/Client.java

```diff
@@ -748,15 +739,15 @@ private boolean testDockstoreTool(Tool tool) {
                             LOG.info("Unknown descriptor, skipping");
                             continue;
                         }
-                        String descriptorPath = DockstoreEntryHelper.convertDockstoreAbsolutePathToJenkinsRelativePath(descriptor.getPath());
+                        String descriptorPath = DockstoreEntryHelper
+                                .convertDockstoreAbsolutePathToJenkinsRelativePath(descriptor.getPath());
                         testParameterFiles = containersApi.getTestParameterFiles(entryId, descriptorType, tagName);
-                        testParameterFiles.stream()
-                                .map(testParameterFile -> DockstoreEntryHelper.convertDockstoreAbsolutePathToJenkinsRelativePath(testParameterFile.getPath()))
-                                .forEach(parameterPath -> {
-                                    parametersList.add(parameterPath);
-                                    descriptorsList.add(descriptorPath);
-                                    commandsList.add(DockstoreEntryHelper.generateLaunchEntryCommand(dockstoreTool, tag, parameterPath));
-                            });
+                        testParameterFiles.stream().map(testParameterFile -> DockstoreEntryHelper
```

Comment:
This looks like the same code as line 617. Reuse?

Reply:
Too much change, I'll save refactoring unrelated code for later